client/src/components/FlightMap.js

```diff
@@ -508,6 +520,22 @@ const FlightPlanMap = props => {
 							})}
 						</LayerGroup>
 					</LayersControl.Overlay>
+					<LayersControl.Overlay checked name="test">
+						<LayerGroup>
+							<Polyline positions={props.getters.planePoints.slice(-20)} color="#8a2be2" weight={3} />
+							{props.getters.planePoints.map((marker, index) => {
+								return popup(marker, index, "uavPath")
```

Comment:
You should make and use custom datatypes - see how `airdropBoundary` is defined above. Also, note the `name={props.display.airdropBoundary[1]}` which solves the previous comment.

Reply:
`props.display` gets you the display name of a certain property. With this comment, I didn't want you to use the airdrop boundary display name for the uav path - I wanted you to create a custom property. You did this, but never gave it a display name and never used it in the `name=` field. So close!